hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleContextImpl.java

```diff
@@ -542,11 +542,13 @@ public <T> T doDispatchPrecedingTransaction(
         if (category != TransactionCategory.USER && category != TransactionCategory.CHILD) {
             throw new IllegalArgumentException("Only user- or child-transactions can dispatch preceding transactions");
         }
+        // This condition fails, because for lazy-account creation we charge fees, before dispatching the transaction,
+        // and the state will be modified.
 
-        if (stack.depth() > 1) {
-            throw new IllegalStateException(
-                    "Cannot dispatch a preceding transaction when a savepoint has been created");
-        }
+        //        if (stack.depth() > 1) {
```

Comment:
Is there a way to retain this check and the one below and only skip for lazy/auto create transactions?

Reply:
It will be triggered for any preceding ChildTransactions. Currently it is just lazy and auto-creations. 🙂